packages/jobs/lib/execution/sync.ts

```diff
@@ -293,32 +303,36 @@ export async function handleSyncSuccess({ nangoProps }: { nangoProps: NangoProps
                         model
                     }
                 });
+
                 void tracer.scope().activate(span, async () => {
-                    try {
-                        const res = await sendSyncWebhook({
-                            connection: connection,
-                            environment: environment!,
-                            webhookSettings,
-                            syncName: nangoProps.syncConfig.sync_name,
-                            model,
-                            now: nangoProps.startedAt,
-                            success: true,
-                            responseResults: {
-                                added,
-                                updated,
-                                deleted
-                            },
-                            operation: lastSyncDate ? SyncType.INCREMENTAL : SyncType.FULL,
-                            logCtx
-                        });
-
-                        if (res.isErr()) {
-                            throw new Error(`Failed to send webhook for sync: ${nangoProps.syncConfig.sync_name}`);
+                    if (team && environment && providerConfig) {
```

Comment:
should we report or log an error if any of this attributes are missing?

Reply:
See 7510a1da5